readthedocs/restapi/views/integrations.py

```diff
@@ -137,8 +137,16 @@ class GitHubWebhookView(WebhookMixin, APIView):
 
         {
             "ref": "branch-name",
+            "head_commit": {
+                "id": "sha",
+                "message": "Update README.md",
+                ...
+            }
             ...
         }
+
+    See full payload here:
+    https://developer.github.com/v3/activity/events/types/#pushevent
```

Comment:
Are we using the v3 of the gh api?

Reply:
It seems, https://developer.github.com/v3/#current-version, since we are not sending the `Accept` header and `v3` is the default.